src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -369,10 +368,10 @@ internal unsafe void SendError(ulong requestId, int httpStatusCode, IList<string
                 }
 
                 httpResponse.Response_V1.StatusCode = (ushort)httpStatusCode;
-                string statusDescription = HttpReasonPhrase.Get(httpStatusCode);
+                string? statusDescription = HttpReasonPhrase.Get(httpStatusCode);
                 uint dataWritten = 0;
                 uint statusCode;
-                byte[] byteReason = HeaderEncoding.GetBytes(statusDescription);
+                byte[] byteReason = statusDescription != null ? HeaderEncoding.GetBytes(statusDescription) : Array.Empty<byte>();
```

Comment:
Seems like this could pass null to `HeaderEncoding.GetBytes` if there is an unexpected status code, or one that has a null reason (e.g. 420).